stock_available_unreserved/models/product.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2018 Camptocamp SA
+# Copyright 2016 ACSONE SA/NV (<http://acsone.eu>)
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models, _
+from odoo.addons import decimal_precision as dp
+
+UNIT = dp.get_precision('Product Unit of Measure')
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    qty_available_not_res = fields.Float(
+        string='Quantity On Hand Unreserved',
+        digits=UNIT,
+        compute='_compute_product_available_not_res',
+    )
+
+    qty_available_stock_text = fields.Char(
+        compute='_compute_product_available_not_res',
+        string='Unreserved stock quantity',
+    )
+
+    @api.multi
+    @api.depends('product_variant_ids.qty_available_not_res')
+    def _compute_product_available_not_res(self):
+        no_new = self.filtered(lambda x: not isinstance(x.id, models.NewId))
+        for tmpl in no_new:
+            tmpl.qty_available_not_res = sum(
+                tmpl.mapped('product_variant_ids.qty_available_not_res')
+            )
+            tmpl.qty_available_stock_text = "/".join(
+                tmpl.mapped('product_variant_ids.qty_available_stock_text')
+            )
+
+    @api.multi
+    def action_open_quants_unreserved(self):
+        products = self.mapped('product_variant_ids').ids
+        quants = self.env['stock.quant'].search(
+            [
+                ('product_id', 'in', products),
+            ]
+        )
+        quant_ids = quants.filtered(
+            lambda x: x.product_id.qty_available_not_res > 0
+        ).ids
+        result = self.env.ref('stock.product_open_quants').read()[0]
+        result['domain'] = "[('id', 'in', {})]".format(quant_ids)
+        result['context'] = "{'search_default_locationgroup': 1, " \
+                            "'search_default_internal_loc': 1}"
+        return result
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    qty_available_not_res = fields.Float(
+        string='Qty Available Not Reserved',
+        digits=UNIT,
+        compute='_compute_qty_available_not_reserved',
+    )
+
+    qty_available_stock_text = fields.Char(
+        compute='_compute_qty_available_not_reserved',
+        string='Available per stock',
+    )
+
+    @api.multi
+    def _product_available_not_res_hook(self, quants):
```

Comment:
seems not used anywhere, which means no hook at all: still needed?

Reply:
got lost during refactoring :face_with_head_bandage: 
will reimplement it )

Reply:
(fixed as reimplemented here https://github.com/OCA/stock-logistics-warehouse/pull/375/files#diff-65190c8c38ef3b3e1094f032c1298735R107)